Jenkinsfile

```diff
@@ -22,8 +35,8 @@ pipeline {
                 stage('Integration Tests') {
                     agent { label 'win10-dservices' }
                     steps {
-                        bat "npm i"
-                        bat "npm run test:int"
+                        bat "npm install"
+                        bat "npm run test:int -- --color=false --no-cache --verbose"
```

Comment:
I don't think these cli args get passed through to jest, unless someone has changed the runner script since I added it.

Reply:
This currently only works in layouts, but these arguments should be getting added to all services once integration tests are added to the tooling (currently WIP).

Reply:
Ok, may as well leave it then.